pkg/riff/commands/stream_delete.go

```diff
@@ -61,9 +63,12 @@ func NewStreamDeleteCommand(c *cli.Config) *cobra.Command {
 	opts := &StreamDeleteOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "delete",
-		Short:   "<todo>",
-		Example: "<todo>",
+		Use:   "delete",
+		Short: "delete a stream of messages",
```

Comment:
What is the effect of attempting to delete a stream which is in use? Does the deletion fail or do the processors using the stream stop working? What about a processor which joins multiple streams when just one of the streams is deleted?
```suggestion
		Short: "delete a stream of messages",
		Long: "Delete a stream of messages or all streams of messages in a namespace.",
```